docs/about.rst

```diff
@@ -36,10 +36,10 @@ process of sufficient complexity, including
  * physical experiments performed in a laboratory environment, and
  * other nontrivial data-generating campaigns.
 
-A simulation's expense can be judged by any of several factors, including
+A simulation\'s expense can be judged by any of several factors, including
```

Comment:
I kind of hate this -- The benefit of md, rst, etc. is that they should be readable when opened in a plain text editor or when viewed in the browser, and this kind of detracts from the former (not much, but still).

But I guess we don't have another choice, and I've kind of given up on looking for a solution online.